### PR TITLE
Updating some wording

### PR DESCRIPTION
**Pull request by sarahframe**

### CODE REVIEW

File: cdhweb/pages/blocks/accordion_block.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ class AccordionTitles(models.TextChoices):
         CODE = "code", "Code"
         DATA = "data", "Data"
         DOCUMENTATION = "documentation", "Documentation"
-        ANALYSIS = "analysis", "Analysis"
+        PUBLICATIONS_AND_PRESENTATIONS = "publications_and_presentations", "Publications and Presentations"
```
Comment (liamjohnston) [inline cdhweb/pages/blocks/accordion_block.py:67]: Do we know what their casing preference is? 
I've been erring on the side of sentence case (e.g. "Publications and presentations") not title case (e.g. "Publications and presentations"). I think sentence case is becoming more common these days.

Comment (haydngreatnews) [reply cdhweb/pages/blocks/accordion_block.py:67]: Looking at the strings we have below this one (`RELATED_COURSES`, `PROJECT_PEER_REVIEW`) it looks like they're all Title Case, and IIRC, there's a style-guide difference -- For what it's worth,[ APA recommends title-case for this situation](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)

Comment (sarahframe) [reply cdhweb/pages/blocks/accordion_block.py:67]: So is this OK to be left as it is? @haydngreatnews @liamjohnston

Comment (haydngreatnews) [reply cdhweb/pages/blocks/accordion_block.py:67]: Yep, good by me

**Review by haydngreatnews — APPROVED** [review]

**Pull request merged**